homeassistant/components/group.py

```diff
@@ -112,6 +113,11 @@ def setup(hass, config):
 
 class Group(object):
```

Comment:
I think mostly that would work. It would require a decent amount of re-factoring to the Group class  though. As a future update, making the group and location entities use the Entity base class would be a good idea. It would definitely be a good practice to have all entities use the Entity base class.